x/oracle/client/cli/query_prices.go

```diff
@@ -48,7 +48,7 @@ import (
 func CmdShowPrices() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "show-prices [token-id]",
-		Short: "shows a prices",
+		Short: "shows all prices of a specific token",
```

Comment:
Maybe say that it's for the latest round?

Reply:
udpated